website/server/models/user/hooks.js

```diff
@@ -59,6 +66,10 @@ function _populateDefaultTasks (user, taskTypes) {
         });
       }
 
+      if (taskDefaults.tags) {
+        newTask.tags = _.map(taskDefaults.tags, findTag);
```

Comment:
what if a tag isn't found in user.tags? We don't want to have task.tags include undefined or null values. Probably you'll want to call `_.compact` on the result https://lodash.com/docs/4.17.4#compact